.kube/base/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tegola
+  labels:
+    app: tegola
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: tegola
+  template:
+    metadata:
+      labels:
+        app: tegola
+    spec:
+      terminationGracePeriodSeconds: 300
+      containers:
+        - name: tegola
+          image: 333778291293.dkr.ecr.us-west-2.amazonaws.com/tegola:confd
```

Comment:
Just checking if this is the right tag (confd).

Reply:
Yes, I updated the base to `hash-handler-config` because I saw there were new commits. I failed to notice they were only merges that [didn't provide any changes](https://github.com/Billups/tegola/compare/confd...hash-handler-config) I've changed the base back to `confd` since that's what's currently deployed to rancher & doesn't require changing these image lines.